pages/getting-started/first-steps-with-docker.mdx

```diff
@@ -0,0 +1,522 @@
+
+Here's how you can manage the Docker service on Windows using PowerShell:
+
+### Open PowerShell as an administrator
+
+- Right-click the Start button and select "Terminal (Admin)" or search
+  for "PowerShell" in the start menu, right-click "Windows PowerShell", and
+  select "Run as administrator".
+
+### Start the Docker Service
+
+If Docker isn't running and you want to start it, use the following command:
+
+```powershell
+net start com.docker.service
+```
+
+You should receive a message indicating that the Docker service has started successfully.
+
+### Stop the Docker Service
+
+To stop the Docker service, use the following command:
+
+```powershell
+net stop com.docker.service
+```
+
+You should receive a message indicating that the Docker service has stopped.
+
+### Check the Status of the Docker Service
+
+To verify the status of the Docker service, whether it's running or stopped, use:
+
+```powershell
+Get-Service com.docker.service
+```
+
+This command will display the status of the Docker service.
+
+## Work with Docker
+
+If you are new to Docker, this guide will help you get a grasp of Docker and
+make it easier to accomplish tasks within Memgraph. After installing Docker, all
+commands are run from the command-line tool of your preference.
+
+### Download the Memgraph Docker image
+
+You can think about downloading Memgraph Docker images as obtaining a blueprint
+for Memgraph, complete with all its core features. This image is your starting
+point. From it, you can launch a container, which acts as a live instance of
+Memgraph. All your interactions, queries, and transactions with Memgraph occur
+within this container. This ensures a consistent and isolated setting for all
+your database tasks. From a single image, you can spawn multiple containers.
+Each container functions as a distinct Memgraph instance.
+
+Images are downloaded using the `docker pull` command followed by the name of
+the Docker image. We encourage you to use Memgraph Platform as it includes
+everything you might need while making the best of Memgraph.
+
+To download the latest Memgraph Platform image, run:
+
+```
+docker pull memgraph/memgraph-platform
+```
+
+All Memgraph Docker images are availble at [Memgraph's Docker
+Hub](https://hub.docker.com/u/memgraph). There you will find exact Docker image
+tags you can downalod and use to install Memgraph with. 
+
+### Architecture of Docker container running Memgraph
+
+The picture below shows the architecture of the Memgraph Docker ecosystem.
+
+![](/pages/getting-started/install-memgraph/docker/docker-architecture.png)
+
+Memgraph offers various Docker images, each consisting of different components
+
+- `memgraph-platform` includes Memgraph database, mgconsole, MAGE and Memgraph Lab
+- `memgraph-mage` includes Memgraph database, mgconsole and MAGE
+- `memgraph includes` Memgraph database and mgconsole
+
+We recommend using the `memgraph-platform` image as it includes all the tools
```

Comment:
If you're new to Memgraph or you're in a developing stage, we recommend using the `memgraph-platform` image as it includes all the tools you might need to analyze your data. Once you're ready to go to production you can choose to use the `memgraph-mage` Docker image that includes the database and the graph library, or run the database only, using the `memgraph` image.

Reply:
Updated.